robot/basestation/static/js/app.js

```diff
@@ -552,3 +552,39 @@ $(document).ready(() => {
     }
   })
 })
+
+function pingMCU(component) {
+  if (millisSince(lastCmdSent) > PING_THROTTLE_TIME) {
+    switch(component)
+    {
+        case "Arm" :
+          sendArmRequest('ping', function (msgs) {})
+          break;
+        case "Rover" :
+          sendRoverRequest('ping', function (msgs) {})
+          break;
```

Comment:
maybe we could just have a ping() function and the default is to ping the odroid (but you could also pass "Odroid" in as a parameter ofc)

Reply:
I don't think it really adds that much value. Plus if we add many other pings we will be stuck with a super large function that is the master of all pings.